Resources/Private/JavaScript/media-module/src/components/Dialogs/UploadDialog.tsx

```diff
@@ -1,273 +1,13 @@
 import * as React from 'react';
```

Comment:
I split up the whole upload dialog to reuse as much parts as possible. The whole upload process might be a candidate to move to ist own package...

Reply:
Done that.